cfgov/alerts/post_sqs_messages.py

```diff
@@ -1,9 +1,9 @@
 import argparse
 import boto3
-import json
 import logging
-import github3
-import requests
+
```

Comment:
Minor request: add `from __future__ import unicode_literals` so that if the body string contains unicode characters it won't raise an exception when used in string formatting below.